src/main/java/com/iwaconsolti/school/config/SchoolPopulatedConfig.java

```diff
@@ -0,0 +1,96 @@
+package com.iwaconsolti.school.config;
+
+import com.iwaconsolti.school.model.Course;
+import com.iwaconsolti.school.model.Grade;
+import com.iwaconsolti.school.model.schools.GerardoInstitute;
+import com.iwaconsolti.school.model.School;
+import com.iwaconsolti.school.model.schools.ZetCollege;
+import com.iwaconsolti.school.model.Student;
+import com.iwaconsolti.school.service.SchoolService;
+import com.iwaconsolti.school.service.SchoolServiceImpl;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.beans.factory.annotation.Value;
+
+
+@Profile("populated")
+@Configuration
+@Slf4j
+public class SchoolPopulatedConfig {
+    @Value("${school.score.limit:100}")
+    private int limitGrade;
+
+    @Bean
+    public School gerardoInstitute() {
+        return new GerardoInstitute();
+    }
+
+    @Bean
+    public School zetCollege() {
+        return new ZetCollege();
+    }
+
+    @Bean("gerardoService")
+    public SchoolService gerardoServicePopulated() {
```

Comment:
Methods should start with a verb describing the action.
Change:
- gerardoServicePopulated()
To:
- createGerardoService()